src/renderer/vt/vtrenderer.hpp

```diff
@@ -195,6 +195,7 @@ namespace Microsoft::Console::Render
         [[nodiscard]] HRESULT _SetInvisible(const bool isInvisible) noexcept;
         [[nodiscard]] HRESULT _SetCrossedOut(const bool isCrossedOut) noexcept;
         [[nodiscard]] HRESULT _SetReverseVideo(const bool isReversed) noexcept;
+        [[nodiscard]] HRESULT _SetHyperlink(const std::wstring uri, USHORT id) noexcept;

```

Comment:
technically these guys should be `const&` if possible
```suggestion
        [[nodiscard]] HRESULT _SetHyperlink(const std::wstring& uri, const USHORT& id) noexcept;
```